tests/rare/ClogRemoteTLog.toml

```diff
@@ -30,13 +31,12 @@ testTitle = "ClogRemoteTLog"
 testName = 'Cycle'
 nodeCount = 30
 transactionsPerSecond = 250.0
-testDuration = 120.0
+testDuration = 360.0
 expectedRate = 0
 
 [[test.workload]]
 testName = 'ClogRemoteTLog'
 lagMeasurementFrequency = 5.0
 clogInitDelay = 10.0
-clogDuration = 110.0
-testDuration = 120.0
+testDuration = 360.0
```

Comment:
I checked SIM_SPEEDUP_AFTER_SECONDS=450. So this clogging happens before simulation clears it, which is a good setting.
```
disableConnectionFailuresAfter(FLOW_KNOBS->SIM_SPEEDUP_AFTER_SECONDS, "Tester");
```